scripts/benchmarks/benchmark.py

```diff
@@ -441,6 +580,7 @@ def maybe_get_experiment_error_traceback(self):
 def prepare_arguments(args):
     defaults = ConfigUtils.read_yaml(args.defaults_config_path)
     defaults["training_data_path"] = args.dataset_save_path
+    defaults["skip_memory_metrics"] = not args.log_memory_hf
```

Comment:
rather than change the defaults, I think you should handle this the same way `"output_dir"` is handled.